packages/report-server/src/reportBuilder/transform/transform.ts

```diff
@@ -6,10 +6,20 @@
 import { yup } from '@tupaia/utils';
 
 import { Row } from '../types';
+import { Context } from '../context';
 import { transformBuilders } from './functions';
 import { aliases } from './aliases';
 
-type TransformParams = {
+export type TransformParams =
+  | keyof typeof aliases
+  | {
+      [key: string]: string | Record<string, string> | undefined;
+      transform: string;
+      title?: string;
+      description?: string;
+    };
```

Comment:
Not major, but I usually find that if I set up the yup validator properly I don't need to explicitly set up a type?

Reply:
Great point! Will use that 👍